networkd_db/networkd_db.ml

```diff
@@ -43,7 +43,10 @@ let _ =
 			if List.mem_assoc !bridge config.bridge_config then begin
 				let bridge_config = List.assoc !bridge config.bridge_config in
 				let ifaces = List.flatten (List.map (fun (_, port) -> port.interfaces) bridge_config.ports) in
-				Printf.printf "interfaces=%s\n" (String.concat "," ifaces)
+				begin match bridge_config.vlan with
+					| None -> Printf.printf "interfaces=%s\n" (String.concat "," ifaces)
+					| Some (parent, id) -> Printf.printf "interfaces=%s vlan=%d parent=%s\n" (String.concat "," ifaces) id parent
```

Comment:
The output of this is program has key/value pairs separated by newlines. We should do that here as well. Furthermore, we always print the interfaces, so it can remain outside the match.